ons_alpha/jinja2/component_overrides/footer/_macro.njk

```diff
@@ -0,0 +1,218 @@
+{% macro onsFooterNew(params) %}{# This is an override of ONS "components/footer/_macro.njk" #}
+    {% from "components/list/_macro.njk" import onsList %}
+    {% from "components/icon/_macro.njk" import onsIcon %}
+
+    {% if params.lang %}
+        {% set lang = params.lang %}
+    {% else %}
+        {% set lang = 'en' %}
+    {% endif %}
+
+    {% set onsLogo %}
+        {% if params.lang == 'cy' %}
+            {{-
+                onsIcon({
+                    "iconType": 'ons-logo-cy',
+                    "altText": 'Swyddfa Ystadegau Gwladol',
+                    "altTextId": 'ons-logo-cy-footer-alt'
+                })
+            -}}
+        {% else %}
+            {{-
+                onsIcon({
+                    "iconType": 'ons-logo-en',
+                    "altText": 'Office for National Statistics',
+                    "altTextId": 'ons-logo-en-footer-alt'
+                })
+            -}}
+        {% endif %}
+    {% endset %}
+
+    <footer class="ons-footer footer">
+        {% if params.footerWarning %}
+            {% from "components/panel/_macro.njk" import onsPanel %}
+            <div class="ons-footer__warning">
+                <div class="ons-container">
+                    {%
+                        call onsPanel({
```

Comment:
Is the `call` bit necessary on this line? Seems a bit out-of-place with all of the other components

Reply:
As mentioned earlier in stand-up, this is basically a copy of the footer code in order to inject a few extra classes - so I don't really want to spend time fixing their code here.